wordpress/wp-content/plugins/cds-base/subscriptions/confirm.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+use NotifyMailer\CDS\NotifyMailer;
+use Ramsey\Uuid\Uuid;
+
+require_once __DIR__ . '/../email/mailer.php';
+
+function cds_subscriptions_send_confirmation_email($data): WP_REST_Response
+{
+    global $wpdb;
+    $notifyMailer = new NotifyMailer();
+
+    // Validate the request contains email and form_id
+    if ($errors = cds_subscriptions_validate_request($data)) {
+        $response = new WP_REST_Response([
+            'errors' => $errors
+        ]);
+
+        $response->set_status(400);
+        return $response;
+    }
+
+    $email = $data['email'];
```

Comment:
Later we'll need to do some validation here

Reply:
there's validation for the existence of properties, but yeah we should validate the types etc too